src/app/_types/metadataTypes.ts

```diff
@@ -4,15 +4,15 @@ type OpenGraphMetadata = {
 }
 
 type TwitterMetadata = {
-  title?: string
-  card?: string
+  card?: 'summary' | 'summary_large_image' | 'player' | 'app'
   site?: string
   creator?: string
 }
 
 export type SeoMetadata = {
   title: string
   description: string
+  image?: { src?: string }
```

Comment:
Could `image` be of type `string` instead of an object?

Reply:
Yes!